Sources/Exporters/Stdout/StdoutExporter.swift

```diff
@@ -32,6 +32,7 @@ public class StdoutExporter: SpanExporter {
             print("ParentSpanId: \(span.parentSpanId?.hexString ?? "no Parent")")
```

Comment:
this PR is fine. Generally though we are trying to output to stdout in a form of JSON in a single line. It is more applicable for server side probably where stdout can be consumed programmatically by fluentBit or something like this

Reply:
Yes, it has much more sense. Created an issue to track this change: #11